src/views/Home/Home.tsx

```diff
@@ -86,31 +90,38 @@ const CTACards = styled(BaseLayout)`
 
 const Home: React.FC = () => {
   const { t } = useTranslation()
+  const { currentLotteryPrize } = useFetchLotteryForPromos()
+  const cakePriceBusd = usePriceCakeBusd()
```

Comment:
On each price change will the home page and its sub components rerendered?

Reply:
Good point mem